src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
```

Comment:
- [ ] I think you are missing the `createStore` module that should be imported from redux as well. This is part of the requirements for this project as you need to create the store using this module, so please make sure to add it to your project.